app/events/views.py

```diff
@@ -172,41 +172,46 @@ def get_user_specific_event(current_user, event_id):
             response.status_code = 404  # Not found
             return response
     else:
-        return jsonify({'Info': "No event created so far"})
+        return jsonify({'Info': "No Events Found"})
 
 
 # Route to display all individual events
 @event.route('/my_events', methods=['GET'])
+@event.route('/my_events/page=<int:page>', methods=['GET'])
 @event.route('/my_events/page=<int:page>&limit=<int:limit>', methods=['GET'])
```

Comment:
you should try using `request.args.get` method instead of hard-coding them in the URL
then you can get them in the code like `page = request.args.get('page', 1, type=int)`
and in the url you can access them using `/api/events/?page=3&limit=4`